setup.sh

```diff
@@ -41,7 +41,7 @@ check() {
     echo BASE_URL=https://webconsole-navigation.svc.${TRITON_ACCOUNT}.${TRITON_DC}.triton.zone >> _env
     echo >> _env
     echo PORT=8080 >> _env
-    echo NODE_ENV=production >> _env
+    echo NODE_ENV=develop >> _env
```

Comment:
.env to be consistent with other environments though that can come as a follow up PR since that wasn't really scoped.